pkg/proto/nbcontract/v1/networkconfig.proto

```diff
@@ -11,10 +11,10 @@ message NetworkConfig {
   NetworkPolicy network_policy = 2;
 
   // URL to the vnet cni plugins tarball.
-  string vnet_cni_plugins_url = 3;
+  reserved 3;
```

Comment:
cool people might do `reserved 3,4` or `reserved 3 to 4` https://protobuf.dev/programming-guides/proto3/#reserved-field-numbers